BondInventoryControl/src/model/dataModel/Inventory.java

```diff
@@ -0,0 +1,43 @@
+package model.dataModel;
+import java.math.BigDecimal;
```

Comment:
他のファイルもですが、
packageの宣言、import、クラスの宣言の間は１行開けましょう。
その方が見やすいです。
```suggestion
package model.dataModel;

import java.math.BigDecimal;

public class Inventory {
```